content/es/working-groups.md

```diff
@@ -0,0 +1,279 @@
+# io.js Grupos de Trabajo
+
+Los Grupos de Trabajo (GT) de io.js son proyectos autónomos creados por el
+[Comité Tecnico (CT)](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#technical-committee).
+
+Los Grupos de Trabajo pueden formarse en cualquier momento pero deben de ser ratificados por el CT.
+Una vez formados, el trabajo definido en la carta del Grupo de Trabajo es responsabilidad del Grupo de Trabajo en lugar del CT.
+
+Es importante que los GT no se formen prematuramente. Los grupos de trabajo no son formados para *comenzar* una serie de tareas si no más bien formados una vez que el trabajo se esta llevando a cabo y los colaboradores piensan que es beneficioso hacerse como un proyecto autónomo.
+
+Si el trabajo definido en la carta del Grupo de Trabajo se completa, el Grupo debe disolverse y la responsabilidad de gobernación absorbida de nuevo por el CT.
+
+## Grupos de Trabajo actuales
+
+* [Website](#website)
+* [Streams](#streams)
+* [Build](#build)
+* [Tracing](#tracing)
+* [i18n](#i18n)
+* [Evangelism](#evangelism)
+* [Roadmap](#roadmap)
+* [Docker](#docker)
+* [Iniciar un Grupo de Trabajo](#starting-a-wg)
+* [Bootstrap Governance](#bootstrap-governance)
+
+### [Website](https://github.com/iojs/website)
+
+El propósito del grupo de trabajo del website es el de construir y mantener un sitio público para el proyecto de `io.js`.
+
+Sus responsabilidades son:
+* Desarrollar y mantener un sistema automatizado para `iojs.org`.
+* Asegurarse de que el sitio se actualiza regularmente con cambios hechos en `io.js`, tales como publicaciones y nuevas funciones.
+* Acoger y habilitar una comunidad de traductores.
+
+### [Streams](https://github.com/iojs/readable-stream)
+
+El GT Streams está dedicado a dar soporte y mejorar la API de Streams como es
+usada en io.js y el sistema npm. Buscamos crear una API capaz de componerse que resuelva el problema presentado por multiples ocurrencias de un evento a lo largo del tiempo de una forma humana y con poca complejidad. Las mejoras en la API vendrán determinadas por las necesidades del ecosistema; interoperabilidad y compatibilidad con otras soluciones y versiones anteriores es de vital importancia. Nuestras responsabilidades incluyen:
+
+* Hacer frente a los stream issues en el io.js issue tracker.
```

Comment:
* We should be consistent in the use of the word `issue`, I pointed this about above in other comment